lib/vyasa_web/live/mode_live/mediator.ex

```diff
@@ -150,6 +165,47 @@ defmodule VyasaWeb.ModeLive.Mediator do
     {:noreply, socket}
   end
 
+  def handle_event(
+        "bind::to",
+        %{"binding" => bind},
+        %Socket{
+          assigns: %{
+            mode: %UserMode{
+              mode_context_component: component,
+              mode_context_component_selector: selector
+            }
+          }
+        } = socket
+      ) do
+
+    {:ok, bind} = Draft.bind_node(bind)
+    # pass binding contexts to the current mode and drafting reflector
+    send_update(component, id: selector, binding: bind)
+    # TODO: implement nav_event handlers from action bar
+    # This is also the event handler that needs to be triggerred if the user clicks on the nav buttons on the media bridge.
+    {:noreply, socket |> UiState.assign(:binding, bind)}
+  end
+
+  def handle_event(
+        "bind::share",_,
+        %Socket{
+          assigns: %{
+            ui_state: %UiState{binding: bind},
+            url_params: %{path: path}
+          }
+        } = socket
+      ) do
+
+    {:ok, shared_bind} = bind
+    |> Draft.create_binding()
+
+    IO.inspect(socket.assigns.url_params)
+
+    {:noreply, socket
+    |> push_event("bind::share", %{url: unverified_url(socket,"#{path}", [bind: shared_bind.id])})
```

Comment:
[because sigil_p has stronger constraints for interpolation i fall back to unverified_url](https://hexdocs.pm/phoenix/Phoenix.VerifiedRoutes.html#unverified_url/3)

Reply:
TIL that sigil_p does more than just nav, it does validation on routing